contracts/modules/TransactionManager.sol

```diff
@@ -214,9 +215,9 @@ abstract contract TransactionManager is BaseModule {
 
     function isAuthorised(address _wallet, address _spender, address _to, bytes calldata _data) internal view returns (bool) {
         if (_to == _spender) {
```

Comment:
Coming back to your comment https://github.com/argentlabs/argent-contracts/pull/187#discussion_r573975163 , I completely agree that this distinction is not a good idea and we should not have restrictions in the `DappRegistry` but let the `Filter` decide for maximum flexibility.

However, in the current implementation of the registry the `getFilter` method checks the existence of a filter for the `_to` address which in the case of an ERC20 method will be the token, while the filter is associated to the `_spender`. So we need to update the logic for these scenarios.